rtloader/doxygen/CustomDoxygen.css

```diff
@@ -0,0 +1,619 @@
+
+/* function descriptions */
```

Comment:
Any specific reason to use a custom css?

Reply:
No particular reason except that I prefer this theme to the base one. I can remove it (and the corresponding configuration option in the Doxyfile) if you think we should remove it

Reply:
It was just a question.